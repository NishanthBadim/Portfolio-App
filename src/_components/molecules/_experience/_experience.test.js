import React from "react";
import { create } from "react-test-renderer";
import "@testing-library/jest-dom/extend-expect";
import Experience from "./_experience";

describe("Experience testing", () => {
  test("Matches the snapshot", () => {
    const input = create(<Experience/>);
    expect(input.toJSON()).toMatchSnapshot();
  })
})