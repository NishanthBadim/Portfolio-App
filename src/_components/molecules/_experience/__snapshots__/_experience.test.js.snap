// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Experience testing Matches the snapshot 1`] = `
<div
  className="mdl-grid"
>
  <div
    className="mdl-cell mdl-cell--4-col"
  >
    <p>
       - 
    </p>
  </div>
  <div
    className="mdl-cell mdl-cell--8-col"
  >
    <h4
      style={
        Object {
          "marginTop": "0px",
        }
      }
    />
    <p />
  </div>
</div>
`;
